examples/stargate-prometheus-grafana/start_stargate-prometheus-grafana.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+#COMPOSE_FILE=stargate-prometheus-grafana.yml
+
+# Make sure backend-1, the seed node, is up before bringing up other nodes and stargate
+docker-compose up -d backend-1
+# Wait until the seed node is up before bringing up more nodes
+(docker-compose logs -f backend-1 &) | grep -q "is now part of the cluster"
+
+# Bring up the 2nd C* node
+(docker-compose logs -f backend-1 &) | grep -q "is now part of the cluster"
+# Bring up the 3rd C* node
+(docker-compose logs -f backend-1 &) | grep -q "is now part of the cluster"
```

Comment:
Should these have corresponding `docker-compose up -d backend-2` and `docker-compose up -d backend-3`?

Reply:
Yes, added those.